### PR TITLE
Ignore fuzzy translations

### LINKED ISSUE #9371: Editor translations do not ignore fuzzy strings
*Bugsquad edit:* Repurposing this issue to track the actual problem: "fuzzy" translation strings are used by the editor, they should not be.

---

**GNU/Linux, Debian 9, 64-bit**

**Issue description**
On deleting an animation of *AnimationPlayer*, a pop-up appears saying "duplicate animation" instead of "delete animation".
Still there in 2.1.4

### PR DESCRIPTION
**Pull request by ghost**
Fixes #9371

### CODE REVIEW

**Comment by akien-mga** [comment]
@r1cebank It seems @Noshyaar beat you to this in the end :)

File: core/io/translation_loader_po.cpp [hunk 1]
```diff
@@ -110,6 +116,9 @@ RES TranslationLoaderPO::load_translation(FileAccess *f, Error *r_error, const S
 		}
 
 		if (l == "" || l.begins_with("#")) {
+			if (l.ends_with("fuzzy")) {
```
Comment (akien-mga) [inline core/io/translation_loader_po.cpp]: There are many kinds of attributes that can go there, not only `fuzzy`, so it's not guaranteed to be the end of the string: https://www.gnu.org/software/gettext/manual/html_node/PO-Files.html
So I don't think you'll get around a `l.find("fuzzy") != -1` here :)

Comment (Noshyaar) [inline core/io/translation_loader_po.cpp]: I've never worked with PO files before. I should have read the spec first, didn't know it had one. I thought it was Weblate specific. :P

**Comment by Geequlim** [comment]
Well done！

**Pull request merged**